doc/docs.md

```diff
@@ -1538,6 +1538,27 @@ println(s)
 // "odd"
 ```
 
+Similar to Rust's `if let`, a value can be bound to a variable within an if statement.
```

Comment:
```suggestion
Anywhere you can use `or {}`, you can also use "if unwrapping", to bound the unwrapped value of an expression, to a variable, when that expression is *not* `none`, nor an error. If unwrapping inverts the control flow for `x := expr or { default }`:
```

Reply:
A good change.

Reply:
This is imho not a sentence that promotes understandability. What about splitting it up?

> Anywhere you can use `or {}`, you can also use "if unwrapping". This binds the unwrapped value of an expression to a variable when that expression is *not* `none` nor an error. 

I'd leave out that it's kind the inverse of error handling with blocks, since this might be confusing. The above sentence already makes clear what it is trying to convey.

Reply:
I agree, shorter sentences are better.